CodeEdit/Utils/Extensions/URL/URL+ResouceValues.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  URL+ResouceValues.swift
+//  CodeEdit
+//
+//  Created by Axel Martinez on 27/6/24.
+//
+
+import Foundation
+import UniformTypeIdentifiers
+
+extension URL {
+    fileprivate var resourceValues: URLResourceValues? {
+        try? self.resourceValues(forKeys: [.isDirectoryKey, .isSymbolicLinkKey, .contentTypeKey])
+    }
+
+    var isFolder: Bool {
+        resourceValues?.isDirectory ?? false
+    }
+
+    var isSymbolicLink: Bool {
+        resourceValues?.isSymbolicLink ?? false
```

Comment:
During my testing I found that aliases were not identified as symbolic links. They are technically different but for the most part functionally the same. This fixes that since we're already grabbing the resource values.
```suggestion
        (resourceValues?.isSymbolicLink ?? false) || (resourceValues?.contentType ?? .item) == .aliasFile
```

Reply:
Awesome! Thanks a lot for the great feedback @thecoolwinter !!